core/resources/scene.yaml

```diff
@@ -110,8 +122,13 @@ layers:
         draw:
             polygons:
                 order: 1
-                color: '#fffffa'
+                #color: '#fffffa'
+                color: global.color.colorA
                 interactive: true # currently ignored
+            lines:
+                order: 2
+                color: function() { return global.color.colorE; }
```

Comment:
Hmm.. I see what you are saying there. (Sorry for the lack of JS knowledge!!).

Will check this.